### ISSUE TITLE
Show/Hide advanced settings in create form not working as expected

### ISSUE THREAD

**Issue by prnam** [kubernetes/dashboard#6668]
##### Environment

```
Installation method: Homebrew
Kubernetes version: latest release
Dashboard version (v1.x is not supported. Use v2.0 or later.): latest release
Operating system: MacOS, Linux and Windows
Node.js version ('node --version' output): 16
Go version ('go version' output): 1.17.5
```

##### Steps to reproduce
Open Dashboard > Click + icon > Select "Create from form" tab > Click "Show advanced settings" > Scroll down to press the same button. It should still have button name "Show advanced settings" instead of "Hide advanced settings"

##### Observed result
Button name is always "Show advanced settings" instead of "Hide advanced settings"

##### Expected result
Although it will hide the advanced settings while clicking "Show advanced settings" it should display  button name as "Hide advanced settings"

##### Comments
PR is raised for the same #6667

**Comment by prnam** [comment]
@floreks, i had also opened this issue. Can you help my link this the PR and close it. I am unable to do myself, hence requesting you.

**Comment by floreks** [comment]
Fixed by #6667

/close
